pkgs/applications/misc/pwsafe/default.nix

```diff
@@ -1,27 +1,28 @@
 { stdenv, fetchFromGitHub, cmake, pkgconfig, zip, gettext, perl
-, wxGTK31, libXext, libXi, libXt, libXtst, xercesc
+, wxGTK30, libXext, libXi, libXt, libXtst, xercesc
 , qrencode, libuuid, libyubikey, yubikey-personalization
-, curl, openssl
+, curl, openssl, file
 }:
 
 stdenv.mkDerivation rec {
   pname = "pwsafe";
-  version = "1.08.2";
+  version = "1.09.0";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
-    rev = "${version}BETA";
-    sha256 = "14qwk3cv5psj7ll71ikyv452x55c7iwjw9765yrpij6741r4yjln";
+    rev = "${version}";
+    sha256 = "0dmazm95d53wq74qvsjvhl7r6fr4dv11nzf8sgdy47nyxv06xs1b";
   };
 
   nativeBuildInputs = [ 
     cmake gettext perl pkgconfig zip
   ];
   buildInputs = [
-    libXext libXi libXt libXtst wxGTK31
+    libXext libXi libXt libXtst wxGTK30
     curl qrencode libuuid openssl xercesc
     libyubikey yubikey-personalization
+    file
```

Comment:
This likely belongs to `nativeBuildInputs` if it is needed only at build time.

Reply:
not the file executable was required but a header file  like include/magic.h

Reply:
ldd pwsafe: 	libmagic.so.1 => /nix/store/d7wqn6pn23m6ijj4qxmnyya7313vx4lc-file-5.38/lib/libmagic.so.1 (0x00007fc146ca9000)

Reply:
fair.